src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -151,6 +155,25 @@ class GrabbitConfiguration {
     }
 
 
+    private static String schemeVal(Map<String, String> configMap, String key,
```

Comment:
Good suggestion, I added a few tests as you suggested. Only in YAML format though, do you think that is sufficient?